src/instructions.spec.ts

```diff
@@ -786,7 +802,7 @@ describe('Cortex-M0+ Instruction Set', () => {
 
   it('should execute a `lsls r5, r5, #18` instruction', async () => {
     await cpu.setPC(0x20000000);
-    await cpu.writeUint16(0x20000000, 0x04ad); // lsls r5, r5, #18
+    await cpu.writeUint16(0x20000000, opcodeLSLSimm(r5,r5,18));// 0x04ad); // lsls r5, r5, #18
```

Comment:
Can you also remove the comment with the opcode at the end of the line? 

We no longer need those since you can see the name and arguments of the opcode in the function call